plugins/hydra_ax_sweeper/README.md

```diff
@@ -0,0 +1,72 @@
+# Hydra Ax Sweeper plugin
+
+This plugin provides a mechanism for Hydra applications to use the [Adaptive Experimentation Platform, aka Ax](https://ax.dev/). Ax can optimize any experiment - machine learning experiments, A/B tests, and simulations. 
+
+Run the following command to install the plugin:
+
+```
+pip install hydra_ax_sweeper
+```
+
+We include an example of how to use this plugin. The file [`plugins/hydra_ax_sweeper/example/banana.py`](plugins/hydra_ax/example/banana.py) implements the [Rosenbrock function (aka Banana function)](https://en.wikipedia.org/wiki/Rosenbrock_function). The return value of the function should be the value that we want to optimize.
+
+To compute the best parameters for the Banana function, clone the code and run the following command in the `plugins/hydra_ax_sweeper` directory:
+
+```
+python example/banana.py -m banana.x=-5:5 banana.y=-5:10.1
```

Comment:
Since we want the user to use floating point for banana function, let's show a correct example.

Reply:
What is wrong in this example? We clearly explain how the parameters are being type casted.

Reply:
For this function, the correct choice is to use floating point numbers. So while the example demos the plugin is gives confusing guidance about this function.

Reply:
Ahh got it :)